SPECS/R/R.spec

```diff
@@ -120,6 +119,8 @@ TZ="Europe/Paris" make check -k -i
 %endif
 
 %changelog
+* Thu Dec 02 2021 Andrew Phelps <anphel@microsoft.com> - 4.1.0-2
```

Comment:
Note: this SPEC was accidentally checked in as Release 2, so I didn't need to increment the release number here.